app/components/tag.tsx

```diff
@@ -1,4 +1,7 @@
 import { BoxProps, Typography, TypographyProps, styled } from "@mui/material";
+import { Theme } from "@mui/material/styles";
+import { makeStyles } from "@mui/styles";
+import cx from "classnames";
```

Comment:
Nit: same as `clsx` we already use in the application? Is so, maybe we could stick to it? 🤔